server/schedule/operator.go

```diff
@@ -104,6 +105,31 @@ func (rp RemovePeer) Influence(opInfluence OpInfluence, region *core.RegionInfo)
 	from.RegionCount--
 }
 
+// SplitRegion is an OperatorStep that splits a region.
+type SplitRegion struct {
+	StartKey, EndKey []byte
+}
+
+func (sr SplitRegion) String() string {
+	return "split region"
+}
+
+// IsFinish checks if current step is finished.
+func (sr SplitRegion) IsFinish(region *core.RegionInfo) bool {
+	return !bytes.Equal(region.StartKey, sr.StartKey) || !bytes.Equal(region.EndKey, sr.EndKey)
```

Comment:
how about we split but merge again, so here the check may still return false?

Reply:
Merge checker will skip this region if it already has an operator.